pyproject.toml

```diff
@@ -39,15 +53,27 @@ pytest-asyncio = "^0.18.3"
 pytest-cov = "^3.0.0"
 liccheck = "^0.7.2"
 pytest-mock = "^3.10.0"
+isort = "^5.11.4"
 
 [tool.poetry.scripts]
 dapp-runner = "dapp_runner.__main__:_cli"
 
-[[tool.poetry.source]]
-name = "testpypi"
-url = "https://test.pypi.org/simple/"
-default = false
-secondary = true
+[tool.poe.tasks]
+checks = {sequence = ["checks_codestyle", "checks_typing", "checks_license"], help = "Runs all available code checks"}
+checks_codestyle = {sequence = ["_checks_codestyle_flake8", "_checks_codestyle_isort", "_checks_codestyle_black"], help = "Runs only code style checks"}
+_checks_codestyle_flake8 = "flake8"
+_checks_codestyle_isort = "isort --check-only --diff ."
+_checks_codestyle_black = "black --check --diff ."
+checks_typing  = {cmd = "mypy --install-types --non-interactive --ignore-missing-imports --check-untyped-defs --warn-unused-ignores --show-error-codes .", help = "Runs only code typing checks" }
+checks_license = {sequence = ["_checks_license_export", "_checks_license_verify"], help = "Runs only license compatibility checks"}
+_checks_license_export = "poetry export -f requirements.txt -o .requirements.txt"
+_checks_license_verify = "liccheck -r .requirements.txt"
+format = {sequence = ["_format_autoflake", "_format_isort", "_format_black"], help = "Runs code auto formatting"}
+_format_autoflake = "autoflake ."
+_format_isort = "isort ."
+_format_black = "black ."
+tests = {sequence = ["tests_ut"], help = "Runs all available types of tests"}
+tests_ut = {cmd = "pytest --cov --cov-report html --cov-report term -sv tests/unit", help = "Runs only unit tests"}
```

Comment:
could we change the name to `tests_unit` (or, even better, `unit_tests`) ? it's just two characters more but reads much more naturally and causes less confusion

Reply:
Here, and in other tasks, we settle down with @lucekdudek to keep same prefix for every sub-command, to be more "hackable" for developers. `tests` here is a prefix, repeated where it could be.

Reply:
@approxit okay, so let's go with `tests_unit`